client/v2/common/common.go

```diff
@@ -103,34 +103,34 @@ func mergeRawQueries(q1, q2 string) string {
 }
 
 // submitForm is a helper used for submitting (ex.) GETs and POSTs to the server
```

Comment:
Can you please fix this comment to match the function name?

Reply:
And does SubmitFormRaw need to be public like this? ("Yes" is a perfectly fine answer)

Reply:
Thanks, I don't think this needs to be public anymore. I was using it before adding "GetRaw"